src/components/Layout/Layout.tsx

```diff
@@ -173,9 +185,9 @@ class Layout extends UIComponent<ReactProps<LayoutProps>, any> {
     if (reducing && isSingleArea) {
       const composedClasses = cx(
         classes.root,
-        startArea && 'ui-layout--reduced__start',
-        mainArea && 'ui-layout--reduced__main',
-        endArea && 'ui-layout--reduced__end',
+        startArea && Layout.slotClassNames.reducedStart,
```

Comment:
Not sure if this is the right convention, as the class here is changed because of a prop...